portfolio/src/main/webapp/index.html

```diff
@@ -12,23 +12,178 @@
   </head>
   <body>
 
-    <div id="header"> 
+    <div class="loader" id="loaderid"></div>
+
+    <div class="hidden" id="header"> 
         <div id="header-spacer"></div>
         <img class="headerels headerimages" src = "images/google.png"/>
-        <p class="headerels">Seun Omonije STEP Portfolio</p>
-        <input id="searchbar" class="headerels" type="text" placeholder="Search about me! (unimplemented)" />
+        <p class="headerels">STEP Portfolio</p>
+        <input id="searchbar" class="headerels" type="text" 
+               placeholder="Learn about me! (try: my name, projects, who I am) Press enter to search" />
+
+        <!-- For some reason i need this in, don't know why-- will use oninput -->
+        <input name="buttonExecute" style="display:none" 
+                onclick="runSearch(document.getElementById('searchbar').value.toLocaleLowerCase())" 
+                type="button" value="Search" />
```

Comment:
sounds like this can be removed now that you listen on onkeydown events

Reply:
I tried to remove it but for some reason it doesn't work

Reply:
It doesn't make any sense...I'm gonna have to look back at the code and figure it out